<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Store - Danh s√°ch Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ecf0f1;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="main_layout">
    <item>
     <layout class="QHBoxLayout" name="header_layout">
      <item>
       <widget class="QLabel" name="title_label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #2c3e50; margin: 20px;</string>
        </property>
        <property name="text">
         <string>üéÆ GAME STORE</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logout_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    padding: 10px 20px;
    font-size: 14px;
    background-color: #e74c3c;
    color: white;
    border: none;
    border-radius: 5px;
    margin: 20px;
}
QPushButton:hover {
    background-color: #c0392b;
}</string>
        </property>
        <property name="text">
         <string>ƒêƒÉng xu·∫•t</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scroll_area">
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="games_widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>796</width>
         <height>69</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="games_layout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
